src/components/Loading/LoadingMessage/LoadingMessage.js

```diff
@@ -16,14 +16,16 @@ const LoadingMessage = ({
   withOverlay,
   ...other
 }) => {
-  const loading = <Loading active={active} small={small} withOverlay={false} />;
+  const loading = (
+    <Loading active={active} small={small} withOverlay={false} {...other} />
+  );
 
   const overlayClasses = classnames('bx--loading-overlay', className, {
     'bx--loading-overlay--stop': !active,
   });
 
   return withOverlay ? (
```

Comment:
Does the `LoadingMessage` not render the `children` (the message) if the overlay is turned off? Shouldn't it always render the message?

Reply:
I'm not sure about this honestly 😅 I did not originally work on this component, and I don't recall what the requirements are/where. Do you mind if I capture this as a separate issue so I can double-check with some folks?

Reply:
Yep! Here it is #444

Reply:
Thank you!